requirements.txt

```diff
@@ -4,6 +4,7 @@ pyflakes
 coverage
 sphinx
 pytest-cov
+nudatus>=0.0.2 
```

Comment:
Hmmm... so this breaks the "just copy the uflash script" for it to just work (tm) feature (important for use in schools where installing dependencies probably isn't an option). :-(